ndoh_hub/settings.py

```diff
@@ -46,7 +46,6 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     # 3rd party
-    "channels",
     "raven.contrib.django.raven_compat",
     "rest_framework",
```

Comment:
There's also a `CHANNEL_LAYERS` value in `settings.py` that we can remove

Reply:
Thanks will remove that as well